src/config/frontend/EdFi.DmsConfigurationService.Frontend.AspNetCore/Modules/IdentityModule.cs

```diff
@@ -97,7 +98,7 @@ bool IsUnique(ClientClientsResult.Success clientSuccess)
 
     private static async Task<IResult> GetClientAccessToken(
         TokenRequest.Validator validator,
-        TokenRequest model,
+        [FromForm] TokenRequest model,
```

Comment:
`[FromForm]` makes the request accept `application/x-www-form-urlencoded` but it stops it from accepting `application/json`. I am afraid a long term solution to accepting both on the same endpoint will require a custom binding, or possibly a classic style MVC controller.

Reply:
We will likely want a custom binding. Introducing MVC controller sounds heavy weight in comparison (I'm assuming that would require additional asp.net related NuGet package(s)?).